Automattic/themes
# Seedlet: Update headstart annotations

This PR partially addresses #2125 to include headstart annotations. It adds a home page layout and one blog post to the user's site, as well as setting the primary and social navigation. 

To test:
- Check out this PR onto your sandbox
- From your sandbox, run: `wp headstart run --user=[dotcom username] --url=[your Headstart test site] --theme=pub/seedlet`
- Visit the site and verify that the headstart content has populated your test site
